module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -1744,6 +1746,18 @@ public function getConfig($function, $params = [])
                 ],
                 'default_sort' => '+due_date',
             ];
+        } elseif ('Holdings' === $function) {
+            $limitByType = $this->config['Holdings']['itemLimitByType'];
+            $type = '';
+            if ($limitByType) {
+                $biblio = $this->getBiblio($params['id']);
+                $type   = $biblio['item_type'];
+            }
+            return [
+                'itemLimit' => $this->config['Holdings']['itemLimitByType'][$type]
```

Comment:
Use `$limitByType` here

Reply:
Thanks @EreMaijala this is included in my followup